### PR TITLE
Install: update to use bootstrap

### PR DESCRIPTION
**Pull request by Williangalvani**
~depends on #42~
needs testing

### CODE REVIEW

File: install/install.sh [hunk 1]
```diff
@@ -40,19 +40,14 @@ echo "Downloading and installing udev rules."
...
-    --network host \
-    -v /dev:/dev \
-    -v /tmp/wpa_playground:/tmp/wpa_playground \
-    -v /var/run/wpa_supplicant/wlan0:/var/run/wpa_supplicant/wlan0 \
-    $COMPANION_CORE
-# Start companion-core for the first time to allow docker to restart it after reboot
-docker start companion-core
+    --name companion-bootstrap \
+    $COMPANION_BOOTSTRAP
+# Start companion-core for the first time to fetch the other images and allow docker to restart it after reboot
```
Comment (patrickelectric) [inline install/install.sh:49]: bootstrap*

File: install/install.sh [hunk 2]
```diff
@@ -40,19 +40,14 @@ echo "Downloading and installing udev rules."
 curl -fsSL $REMOTE/install/udev/100.autopilot.rules -o /etc/udev/rules.d/100.autopilot.rules
 
 echo "Downloading companion-core"
```
Comment (patrickelectric) [inline install/install.sh:42]: bootstrap*

**Review by patrickelectric — CHANGES_REQUESTED** [review]

**Review by patrickelectric — CHANGES_REQUESTED** [review]
Corrections are still necessary in the base image.

File: install/install.sh [hunk 3]
```diff
@@ -39,20 +39,17 @@ echo "Going to install companion-docker version ${VERSION}."
...
-    -v /dev:/dev \
-    -v /tmp/wpa_playground:/tmp/wpa_playground \
-    -v /var/run/wpa_supplicant/wlan0:/var/run/wpa_supplicant/wlan0 \
-    $COMPANION_CORE
-# Start companion-core for the first time to allow docker to restart it after reboot
-docker start companion-core
+    --name companion-bootstrap \
+    -v $HOME/.config/companion:/root/.config/companion \
+    -v /var/run/docker.sock:/var/run/docker.sock \
+    -e COMPANION_CONFIG_PATH=$HOME/.config/companion \
```
Comment (patrickelectric) [inline install/install.sh:51]: Will this variable be accessible in core ?

Comment (Williangalvani) [reply install/install.sh:51]: no

Comment (jaxxzer) [reply install/install.sh:51]: maybe it will be nice to add an exports list to the core config file. here I think we agree this doesn't need to hold

**Review by patrickelectric — APPROVED** [review]
It works

**Pull request merged**